### DISCUSSION TITLE
Problem starting with template variable set

### DISCUSSION THREAD

**Discussion by neoscaler** [jpanther/congo#516] [General]
Hi there,

yesterday I tried to use Congo with a fresh Hugo 0.111.3 installation via module. I exactly followed instructions multiple times, but as soon as I copied over the config.toml Hugo would not start with an error like "module congo not found".

I fixed it after long try'n'error by removing the line 'template = "congo"' from the config.toml.

Is this something I should correct in a PR for the exampleSite/config.toml? Or should we at least add it to the documentation? I am wondering why I am the only one encountering this.

**Comment by jpanther** [comment]
If you follow the [installation instructions](https://jpanther.github.io/congo/docs/installation/#set-up-theme-configuration-files) you should never encounter this problem. As per the docs, you should be copying the configuration from `themes/congo/config/_default` and NOT from the exampleSite directory. The example site configuration is set up specifically for the example site and does not guarantee a correct out of the box configuration.
